docs/tutorials/index.md

```diff
@@ -1,8 +1,9 @@
 ---
 seo:
-    description: The XRP Ledger tutorials walk you through the steps to learn and get started with the XRP Ledger and to use the ledger for advanced use cases.
+    title: Crypto Wallet and Blockchain Development Tutorials
+    description: Learn the basics on how to get started building on the XRP Ledger with these helpful crypto wallet and blockchain tutorials for developers.
```

Comment:
The tutorials include both beginner and advanced topics, so recommend updating the description to reflect that.

Reply:
i removed "the basics on"